pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -2072,3 +2081,15 @@ func LegacySaveCommandToUnstructured(cmd *dashboards.SaveDashboardCommand, names
 
 	return finalObj, nil
 }
+
+func getFolderIds(response *dashboardv0alpha1.SearchResults) []string {
+	folderIds := []string{}
+	folderSet := map[string]string{} // to avoid duplicates
+	for _, hit := range response.Hits {
```

Comment:
This will panic if the response is nil - could you add a check for this?

Reply:
updated.  not passing the response anymore

Reply:
The response can still be nil, though, when you're calling the function. So you'll need to check if response is nil before you access response.Hits